.github/workflows/smartbear-pull-request-comment-created.yaml

```diff
@@ -0,0 +1,38 @@
+name: Pull Request Comment Created
+
+# Once the code author commits all changes to branch, and PR updated, 
+# they can comment with "/done"
+
+on:
+  issue_comment:                                     
+    types: [created]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.issue.pull_request && contains(github.event.comment.body, '/done') }}
+    name: Transition Jira Ticket to In Review
+    steps:
+    - name: login
+      uses: atlassian/gajira-login@v3
+      env:
+        JIRA_BASE_URL: ${{ secrets.JIRA_BASE_URL }}
+        JIRA_USER_EMAIL: ${{ secrets.JIRA_USER_EMAIL }}
+        JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
+
+    - run: echo ${{ github.event.issue.title }}
+
+    - name: Find Issue
+      id: find
+      uses: atlassian/gajira-find-issue-key@v3
+      with:
+        string: ${{ github.event.issue.title }}
+
+    - run: echo "${{ steps.find.outputs.issue }}"
+
+    - name: Transition issue
+      if: steps.find.outputs.issue != ''
+      uses: atlassian/gajira-transition@v3
+      with:
+        issue: ${{ steps.find.outputs.issue }}
+        transition: "In Progress"
```

Comment:
Is this transition correct? the title says " In Review". Which we don't have yet, do we?

Reply:
We don't :( hence using the "In Progress" which will do nothing, its more future proof-ness if we decide to go that way

Reply:
The idea behind this, would be once the author has committed all post-code review changes, he can comment on the ticket "/done" and it will progress the ticket to "in review" in jira (which can trigger other things down the chain)